pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -515,19 +544,19 @@ type EventRateLimitConfigItem struct {
 }
 
 type OPAIntegrationSettings struct {
```

Comment:
Can we add a comment that at least once in our codebase we explain what "OPA" actually means? Like

> OPAIntegrationSettings configure the usage of the Open Policy Agent inside the user cluster.

Reply:
Sure, will add a comment explaining it as ~~Outer Planet Alliance~~ Open Policy Agent.